diagnostic_tool/kinematics_cpp/kinematics.cpp

```diff
@@ -0,0 +1,443 @@
+/*
+ * Adapted from https://github.com/Le0nX/ModernRoboticsCpp
+ */
+#include <cmath>
+#include <vector>
+#include <Eigen/Dense>
+#include <pybind11/pybind11.h>
+#include <pybind11/eigen.h>
+
+namespace py = pybind11;
+
+#define M_PI           3.14159265358979323846  /* pi */
+
+namespace mr {

```

Comment:
would recommend using more meaningful namespace rather than short names like this.

Reply:
Changed to `kincpp`.